class Solution {
    public List<Integer> getRow(int rowIndex) {
        int n = rowIndex+1;
        List<List<Integer>> ans = new ArrayList<>();
        for(int i=0;i<n;i++){
            ans.add(new ArrayList<Integer>());
            for(int j=0;j<=i;j++){
                if(j==0 || j==i){
                    ans.get(i).add(j,1);
                }
                else{
                    ans.get(i).add(ans.get(i-1).get(j)+ans.get(i-1).get(j-1));
                }
            }
        }
        return ans.get(rowIndex) ; 
    }
}